build.sbt

```
@@ -6,15 +6,25 @@ ThisBuild / semanticdbEnabled := true
 ThisBuild / semanticdbVersion := scalafixSemanticdb.revision // use Scalafix compatible version
 ThisBuild / scalafixScalaBinaryVersion := V.scala213BinaryVersion
 ThisBuild / scalafixDependencies ++= List("com.github.liancheng" %% "organize-imports" % V.organizeImports)
+ThisBuild / organization := "ch.epfl.scala"
+ThisBuild / version := "0.1.0-SNAPSHOT"
 
-lazy val interfaces = project
-  .in(file("interfaces"))
+lazy val `compiler-interfaces` = project
+  .in(file("interfaces/compiler"))
   .settings(
     libraryDependencies ++= Seq("ch.epfl.lamp" % "dotty-compiler_0.27" % V.dotty),
```

So that we don't need to update the `_0.27` suffix when we upgrade the Scala 3 version
```suggestion
  .settings(
    scalaVersion := V.dotty,
    libraryDependencies ++= Seq("ch.epfl.lamp" %% "dotty-compiler" % scalaVersion.value),
```